src/sss_iface/sss_iface.c

```diff
@@ -148,6 +136,27 @@ sss_monitor_service_init(TALLOC_CTX *mem_ctx,
     return ret;
 }
 
+errno_t
+sss_monitor_provider_init(struct sbus_connection *conn,
+                          const char *svc_name,
+                          uint16_t svc_version,
+                          uint16_t svc_type)
+{
+    struct tevent_req *req;
+
```

Comment:
Why '_provider_init()' doesn't require `sss_iface_connect_address()`?

Reply:
Providers connects earlier. But I think it can be put here.

Reply:
TODO